### PR TITLE
Elements tab support for iOS Chrome DevTools

### PR DESCRIPTION
**Pull request by Pip3r4o**
The changes in the PR aim to provide implementation for the DOM and CSS agents in the Chrome Devtools Inspector protocol for iOS. 

The implementation steps on top of what was already established in the tns-core-modules to build `dom-node` objects on demand when the Inspector is attached. Thanks to that, no changes to the core modules have been done, but rather only in the `debugger` module.

### The commit message references a specific issue in this repo
Implements https://github.com/NativeScript/ios-runtime/issues/791

### You have [unit tests](https://github.com/NativeScript/NativeScript/blob/master/running-tests.md)
Tried to accommodate the `dom-node-tests` to work with the iOS implementation of the DevTools Inspector.

### CODE REVIEW

**Comment by Pip3r4o** [comment]
run ci

**Comment by Pip3r4o** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

File: tns-core-modules/ui/core/properties/properties.ts [hunk 1]
```diff
@@ -1302,5 +1302,10 @@ export function getComputedCssValues(view: ViewBase): [string, any][] {
     for (var prop of cssPropertyNames) {
         result.push([prop, style[prop]]);
     }
+
+    result.push(["top", "auto"]);
```
Comment (vakrilov) [inline tns-core-modules/ui/core/properties/properties.ts]: Add a comment why these are needed

**Pull request merged**